.travis.yml

```diff
@@ -1,10 +1,6 @@
 language: go
 
 go:
-  - "1.5"
-  - "1.6"
-  - "1.7"
-  - "1.8"
-  - "1.9"
   - "1.10"
+  - "1.11"
```

Comment:
If this is just for us is there any need to test on 1.10?

Reply:
Might as well - it's the earliest version this code will work on AFAIK because we're using strings.Builder which arrived in 1.10.